From: Rafael J. Wysocki <rjw@suse.de>
Subject: x86: Rename display_cacheinfo() to cpu_detect_cache_sizes()
Patch-mainline: n/a

Fix Xen build issue introduced by patch
x86-cpu-mv-display_cacheinfo-cpu_detect_cache_sizes.patch that
renamed display_cacheinfo() to cpu_detect_cache_sizes() in accordance
with the analogous mainline changes, but didn't update Xen.

Signed-off-by: Rafael J. Wysocki <rjw@suse.de>
---
 arch/x86/kernel/cpu/common-xen.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: linux-2.6.32-master/arch/x86/kernel/cpu/common-xen.c
===================================================================
--- linux-2.6.32-master.orig/arch/x86/kernel/cpu/common-xen.c
+++ linux-2.6.32-master/arch/x86/kernel/cpu/common-xen.c
@@ -69,7 +69,7 @@ void __init setup_cpu_local_masks(void)
 static void __cpuinit default_init(struct cpuinfo_x86 *c)
 {
 #ifdef CONFIG_X86_64
-	display_cacheinfo(c);
+	cpu_detect_cache_sizes(c);
 #else
 	/* Not much we can do here... */
 	/* Check if at least it has cpuid */
@@ -419,7 +419,7 @@ static void __cpuinit get_model_name(str
 	}
 }
 
-void __cpuinit display_cacheinfo(struct cpuinfo_x86 *c)
+void __cpuinit cpu_detect_cache_sizes(struct cpuinfo_x86 *c)
 {
 	unsigned int n, dummy, ebx, ecx, edx, l2size;
 
